Lighthouse-Tests.md

```diff
@@ -78,3 +78,36 @@ def export_url(url):
       ]
     },
 ```
+# How to debug Lighthouse tests?
+- **Common question:** What to do if your changes cause the lighthouse accessibility tests to fail?
+- The lighthouse logs will provide you most of the information you need (To see the logs, click on the details button on the right)
+
+## Steps solve the failing lighthouse test
+
+**1. Find out which pages are failing:** You can find which pages are failing in the lighthouse logs
+<img height="150" src="https://github.com/Patel-Muhammad/oppia-web-developer-docs/assets/96219910/a7833e61-8294-446e-a588-bdfbef13ab10">
+
+**2. Find report for failing page:** The lighthouse checks will upload reports to a temporary url for each page that it audits. 
+The lighthouse checks will print the link to each report in the logs.
+
+![image](https://github.com/Patel-Muhammad/oppia-web-developer-docs/assets/96219910/8e5a712c-5409-4e5c-90b6-7c2bb9ad21c3)
```

Comment:
Hi @Patel-Muhammad, all the images seem to be in your fork. Can you add them to the oppia/oppia-web-developer-docs repo instead?

Reply:
Hi @Ash-2k3 can you please let me know how can I do that? Should I add the images to the images folder that is present?

Reply:
Yes that is right, also this is not the first time we are adding images in the wiki. You may take a look at wikis which already has images and mimic the method of adding images.

Reply:
Hi @Ash-2k3 I have done that PTAL!